apps/index.html

```diff
@@ -7,5 +7,14 @@
     <script src="bundle/common.js"></script>
     <script src="bundle/monorail.js"></script>
   </head>
-  <body></body>
+  <body>
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+      ga('create', 'UA-71461226-1', 'auto');
```

Comment:
Hey Roland,

I made a new tracking number specifically for the WebUI - base location expected to be `http://rackhd.github.io`. The code for this property is `UA-69712167-3`